core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+.fa {
+    display: inline-block;
+    font: normal normal normal 14px/1 FontAwesome;
+    padding-right: 4px;
+    font-size: 23px;
+    text-rendering: auto;
+    position: relative;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    top: -10px;
+}
+  
+</style>
   <div class="oppia-footer-social">
     <div>
       <h4>
         <span translate="I18N_FOOTER_FOLLOW_US"></span>:
       </h4>
     </div>
     <a href="https://plus.google.com/109898456505810251700/about" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/gplus.png')]>" alt="Google+">
+      <i class="fa fa-google-plus-square" aria-hidden="true"></i>
```

Comment:
No, I don't think that's necessary and I would remove it from here and below. However, what is necessary is to include the oppia accessibility fix for icons. 

@shaz13 In order to ensure that icons are screenreader accessible, we need to include a label. The fix that we have implemented is to use a span with a special class that will allow the screenreader to read labels for icons.
It looks like this:
`<span class="oppia-icon-accessibility-label">Delete this list entry</span>`

And you can see more about the implementation in PR #3118 

So when you switch out a graphic with an alt attribute for an icon, you will need to use a span like the one above with appropriate text (which should be taken from the alt attribute). The span should go directly beneath the icon. Does that make sense?